core/src/main/java/com/linecorp/armeria/common/util/Holder.java

```diff
@@ -0,0 +1,63 @@
+/*
+ * Copyright 2018 LINE Corporation
+ *
+ * LINE Corporation licenses this file to you under the Apache License,
+ * version 2.0 (the "License"); you may not use this file except in compliance
+ * with the License. You may obtain a copy of the License at:
+ *
+ *   https://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing, software
+ * distributed under the License is distributed on an "AS IS" BASIS, WITHOUT
+ * WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied. See the
+ * License for the specific language governing permissions and limitations
+ * under the License.
+ */
+package com.linecorp.armeria.common.util;
+
+import java.util.function.Supplier;
+
+/**
+ * Mutable object value holder.
+ *
+ * @param <T> Type of value.
+ */
+public final class Holder<T> {
```

Comment:
Could you:

- move this class to `server.annotation`
- make this class package-local
- rename this class to `MutableHolder` to emphasize its mutability?

Reply:
fixed